### PR TITLE
[11.0][ADD] ddmrp_report_part_flow_index

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migrated from Eficent/ddmrp

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Without plurals in module name, please.

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza only the PR and branch names are wrong, the module was already renamed. Fixing it!

**Comment by pedrobaeza** [comment]
OK, thanks

**Review by carlosdauden — APPROVED** [review]
Code LGTM

File: ddmrp_report_part_flow_index/reports/report_ddmrp_part_plan_flow_index.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    flow_index_group_id = fields.Many2one(
+        'ddmrp.flow.index.group', string="Flow Index Group", readonly=True)
+
+    @api.model_cr
+    def init(self):
+        tools.drop_view_if_exists(self._cr,
+                                  'report_ddmrp_part_plan_flow_index')
+        self._cr.execute(
+            """
+            CREATE or REPLACE VIEW report_ddmrp_part_plan_flow_index AS (
```
Comment (pedrobaeza) [inline ddmrp_report_part_flow_index/reports/report_ddmrp_part_plan_flow_index.py]: For proper possible inheritance, split this into smaller methods for select, from, etc.

**Review by pedrobaeza — COMMENTED** [review]
Not really requirement but strongly advised. See my inline comment.

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza could you have a look to my last commit?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**